nx/source/services/ncm.c

```diff
@@ -463,6 +463,56 @@ Result ncmContentMetaDatabaseListContentInfo(NcmContentMetaDatabase* db, const N
     return rc;
 }
 
+Result ncmContentMetaDatabaseList(NcmContentMetaDatabase* db, u32 titleType, u64 titleIdExact, u64 titleIdLow, u64 titleIdHigh, NcmMetaRecord* metaRecordsOut, size_t metaRecordsBufSize, u32* numEntriesWritten, u32* numEntriesTotal) {
+    IpcCommand c;
+    ipcInitialize(&c);    
+    ipcAddRecvBuffer(&c, metaRecordsOut, metaRecordsBufSize, BufferType_Normal);
+    
+    struct {
+        u64 magic;
+        u64 cmd_id;
+        u32 titleType;
+        u64 TID;
```

Comment:
These should match the input params.

Reply:
the FULL CAPS are because thats what they are named on switchbrew wiki, so just rename to match parameters ?

Reply:
yes